### PR TITLE
ADEN-5367 | Remove old openx bidder

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5367

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/evolve2.js [hunk 1]
```diff
@@ -6,8 +6,7 @@ define('ext.wikia.adEngine.provider.evolve2', [
 	'ext.wikia.adEngine.utils.adLogicZoneParams',
 	'ext.wikia.adEngine.utils.eventDispatcher',
 	'wikia.log',
-	require.optional('ext.wikia.adEngine.lookup.openx.openXBidderHelper')
-], function (adContext, gptHelper, slotTweaker, zoneParams, eventDispatcher, log, openXHelper) {
+], function (adContext, gptHelper, slotTweaker, zoneParams, eventDispatcher, log) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/provider/evolve2.js:9]: MAJOR Function has 6 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/AdEngine/js/provider/directGpt.js [hunk 2]
```diff
@@ -7,10 +7,9 @@ define('ext.wikia.adEngine.provider.directGpt', [
 	'ext.wikia.adEngine.slot.adUnitBuilder',
 	'ext.wikia.adEngine.slot.service.kiloAdUnitBuilder',
 	'ext.wikia.adEngine.slotTweaker',
-	require.optional('ext.wikia.adEngine.lookup.openx.openXBidderHelper'),
 	require.optional('ext.wikia.aRecoveryEngine.pageFair.recovery'),
 	require.optional('ext.wikia.aRecoveryEngine.sourcePoint.recovery')
-], function (adContext, uapContext, factory, adUnitBuilder, kiloAdUnitBuilder, slotTweaker, openXHelper, pageFair, sourcePoint) {
+], function (adContext, uapContext, factory, adUnitBuilder, kiloAdUnitBuilder, slotTweaker, pageFair, sourcePoint) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/provider/directGpt.js:12]: MAJOR Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 55 issues
* ![MAJOR][MAJOR] 45 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 6 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L95](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L95): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L96](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L96): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L175](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L175): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L200](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L200): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L200](https://github.com/Wikia/app/blob/a14c08b3a9b6b96fd95574767f850e81b8bd9e1c/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L200): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 61 issues
* ![MAJOR][MAJOR] 50 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 7 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L95](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L95): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L96](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L96): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L175](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L175): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L200](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L200): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L200](https://github.com/Wikia/app/blob/6b061c5afc7933821a04bd40e0cc4aae8f98a964/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L200): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by nandy-andy** [comment]
Do you think it's safe to merge it and release in one go? How about releasing first removal of [inlined scripts](https://github.com/Wikia/app/pull/13116/files#diff-1075e776245fd07de54c50057f532d83) and then removing the assets?

**Comment by kvas-damian** [comment]
I believe it's safe. We already don't call js code (it's disabled in if conditions) and module is in `optionalRequire`.

If you're worried about it we can split the release into two parts.

**Review by nandy-andy — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 61 issues
* ![MAJOR][MAJOR] 50 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 7 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$title". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$article". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$output". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$user". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L19](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L19): Remove the unused function parameter "$wiki". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L95](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L95): Remove the unused function parameter "$scripts". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L96](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L96): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L175](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L175): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L200](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L200): Remove the unused function parameter "$jsExtensionPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. ![MAJOR][MAJOR] [AdEngine2Hooks.class.php#L200](https://github.com/Wikia/app/blob/9410df6d765014a3c8b83669a6272d39622a1753/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L200): Remove the unused function parameter "$scssPackages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**